src/types/query.rs

```diff
@@ -0,0 +1,64 @@
+use edn_rs::Serialize;
+
+#[derive(Clone)]
+pub struct Query {
+    find: Vec<String>,
+    whr: Option<Vec<String>>,
+}
+
+impl Query {
+    pub fn find(find: Vec<&str>) -> Self {
+        Self {
+            find: find
+                .into_iter()
+                .map(|f| String::from(f))
+                .collect::<Vec<String>>(),
+            whr: None,
+        }
+    }
+
+    pub fn where_clause(mut self, whr: Vec<&str>) -> Self {
```

Comment:
I think the `_clause` suffix is unnecessary and verbose, what do you think of `where` directly?

Reply:
we can't call a function `where`

Reply:
💩

Reply:
b9d493b

Reply:
Do you think `where_` is too bad?

Reply:
Nope!

Reply:
We discussed outside here to keep `where_clause` for now, maybe we will change it later